charts/nri-bundle/requirements.lock

```diff
@@ -7,7 +7,7 @@ dependencies:
   version: 1.8.2
 - name: nri-metadata-injection
   repository: file://../nri-metadata-injection
-  version: 1.5.2
+  version: 1.5.3
```

Comment:
Could you bump the requirements.yml as well for metadata injection ?

Reply:
Done